npm-packages/pyoes/scss/pyoes/_placeholder.scss

```diff
@@ -0,0 +1,19 @@
+input::placeholder {
```

Comment:
die input best overal bij zetten of weglaten, kan verwarrend worden anders bij het debuggen